File: ThreeNo_Multiplication/Mult.py
"""
    Author : Pakshal Jain
    Date : 12/10/2022
    Program : To Calcualte Multiplication of Entered Three Numbers
"""

#####################################################################################################################################

#Class Declaration
class Calculate :

    #Characteristics of Class
    No = 0;
    No2 = 0;
    No3 = 0;
    
    #Constructor of Class
    def __init__(self,no,no2,no3) :
        self.No = no;
        self.No2 = no2;
        self.No3 = no3;
    
    #Behaviour of Class
    def Multi(self) :
        ans = 0;

        #Handeling Condition If any of The Entered Number is Zero
        if((self.No == 0)and(self.No2 == 0)) :
            ans = self.No3;
        elif((self.No2 == 0)and(self.No3 == 0)) :
            ans = self.No;
        elif((self.No == 0)and(self.No3 == 0)) :
            ans = self.No2;
        elif(self.No == 0) :
            ans = self.No2 * self.No3;
        elif(self.No2 == 0) :
            ans = self.No * self.No3;
        elif(self.No3 == 0) :
            ans = self.No * self.No2;
        #Handeling Condition if Any of The Entered Number is Not Zero
        else :
            ans = self.No * self.No2 * self.No3;
        
        return ans;

#################################################################################################################################################

def main() :
    No = 0;
    No2 = 0;
    No3 = 0;
    iret = 0;

    print("Jay Ganesh...");

    print("Enter First Number");
    No = int(input());

    print("Enter Second Number");
    No2 = int(input());

    print("Enter Third Number");
    No3 = int(input());

    #Creating object of Calculate Class
    cobj = Calculate(No,No2,No3);

    if((No == 0)and(No2 == 0)and(No3 == 0)) :
        iret = No * No2 * No3;
    else :
        #Calling Behaviour of Class
        iret = cobj.Multi();

    #Printing Final Output    
    print("Multiplication of Three Numebers is : ",iret);

############################################################################################################################################

if __name__ == "__main__" :
    #Calling Main Function
    main();

File: ThreeNo_Multiplication/Mult.java
/*
    Auhtor : Pakshal Jain
    Date : 12/10/2022
    Program : To Calculate Multiplication of Entered Three number
*/

//Required Libraries
import java.lang.*;
import java.util.*;

//Class Declaration
class Calculate
{
    //Characteristics of Class
    int No = 0;
    int No2 = 0;
    int No3 = 0;

    //Constructor of Class
    Calculate(int no,int no2,int no3)
    {
        this.No = no;
        this.No2 = no2;
        this.No3 = no3;
    }

    //Behaviour of Class
    int multiplication()
    {
        int ans = 0;
        
        //Logic
        //Handeling Condtion if Any of Entered Numbers is Zero
        if((this.No == 0)&&(this.No2 == 0))
        {
            ans = this.No3;
        }
        else if((this.No2 == 0)&&(this.No3 == 0))
        {
            ans = this.No;
        }
        else if((this.No == 0)&&(this.No3 == 0))
        {
            ans = this.No2;
        }
        else if(this.No == 0)
        {
            ans = this.No2 * this.No3;
        }
        else if(this.No2 == 0)
        {
            ans = this.No * this.No3;
        }
        else if(this.No3 == 0)
        {
            ans = this.No * this.No2;
        }
        //Handeling Condition if Entered Numbers are Not Zero
        else 
        {
            ans = this.No * this.No2 * this.No3;
        }

        return ans;
    } 
} //End of Class

//Main Class Declaration
class Mult 
{
    public static void main(String args[])
    {
        int No = 0;
        int No2 = 0;
        int No3 = 0;
        int iret = 0;

        System.out.println("Jay Ganesh...");

        //Creating Object of Scanner Class To take input from User
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter First number");
        No = sobj.nextInt();

        System.out.println("Enter Second Number");
        No2 = sobj.nextInt();

        System.out.println("Enter Third Number");
        No3 = sobj.nextInt();

        //Creating Object of Calcualte Class
        Calculate cobj = new Calculate(No,No2,No3);

        //Handeling Condition All Entered Numbers are Zero
        if((No == 0)&&(No2 == 0)&&(No3 == 0))
        {
            iret = No * No2 * No3;
        }
        else 
        {
            //Calling Behaviour of Class
            iret = cobj.multiplication();
        }

        //Printing Final Output
        System.out.println("Multiplication of Entered Three Numbers is : "+iret);
    }
}